core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2945,6 +2948,130 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleUpdateFeatures(request: RequestChannel.Request): Unit = {
```

Comment:
Could we add some unit tests in `KafkaApisTest.scala`, once the refactoring is finished?

Reply:
Seems not covered yet

Reply:
This does not seem to be required, since it is already achieved via `UpdateFeaturesTest`. Infact there we test using admin client, which is even better as it tests e2e client to server functionality.

What do we gain by adding the additional tests in `KafkaApisTest` ?